<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtobusListesi</class>
 <widget class="QWidget" name="OtobusListesi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="grpArama">
       <property name="title">
        <string>Alan seçimi</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbOtobusModel">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOtobusKapasite">
          <property name="text">
           <string>Kapasite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOtobusRenk">
          <property name="text">
           <string>Renk</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOtobusBoyut">
          <property name="text">
           <string>Boyut</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOtobusOturmaDuzeni">
          <property name="text">
           <string>Oturma Düzeni</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOtobusPlaka">
          <property name="text">
           <string>Plaka</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Arama Türü</string>
       </property>
       <widget class="QRadioButton" name="rbBiten">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ile biten</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbIceren">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>içeren</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbBaslayan">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ile başlayan</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Kriter</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Arama Metni</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leAranan">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAra">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>211</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Ara</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tblOtobusler"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leAranan</sender>
   <signal>textChanged(QString)</signal>
   <receiver>btnAra</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
